pset-iterators.js

```diff
@@ -93,6 +105,11 @@ console.log('Problem 3:')
 
 // Add your code below this line
 
+const presidentialParties = presParties.filter(function(presParty){
+  return presParty.party
+})
+console.log(presidentialParties)
+
```

Comment:
@ChicoMangusta  Good job here on this assignment. I see you're using the filter method to return the president party. 
- currently, you are using` presParties ` to run a filter on but the variable presParties is undefined so that would throw an error. if you change the `presParties` to `presidents` your solution should work.
 - Also, the `map` method would've been a good option for this use case as map would've returned all of the parties for you in a new array.

Take a look at this approach [here](https://github.com/jsr20190826/hw-03-iterators-solution/blob/96bab20deba3ad116374a5827291a951faf5d555/pset-iterators.js#L94)